src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrHorizon.hpp

```diff
@@ -43,12 +43,25 @@ namespace gr::Solutions {
  * \f]
  * where the angles are encoded in \f$\hat x\f$ and everything else on the
  * right-hand side is constant.
+ *
+ * The Kerr-Schild radius can be Lorentz boosted in an arbitrary direction
+ * \f$\vec{\beta}\f$. We apply a Lorentz transformation on the Cartesian
+ * Kerr-Schild coordinates and find that the boosted Kerr-Schild radius \f$r'\f$
+ * as a function of angles satisfies
+ * \f[
+ * r'^2 = r^2 (1 + \gamma^2 \left( \beta_x\cos\phi\sin\theta +
```

Comment:
Are these angles in the boosted or unboosted frame? I think they should be in the boosted frame (see comment in the test below).